vault/blog.2022.2022-05-02-self-contained-vaults-release.md.md

```diff
@@ -0,0 +1,72 @@
+---
+id: g98s3m6jft4gnb546y7i4xg
+title: 2022 05 02 Self Contained Vaults Release
+desc: ''
+updated: 1651527766161
+created: 1651526168935
```

Comment:
manually add a date here`eg: date: '2022-04-05'` 
this is so that publishing can sort this properly